data/outfits.txt

```diff
@@ -397,6 +397,12 @@ outfit "Local Map"
 	"map" 12
 	description "This data chip contains complete information on the twelve star systems closest to this one: planets, ports, governments, trade prices, available services, etc. You can get all the same information just by exploring those systems yourself, but having a map can save you from making wrong turns if you are trying to travel through new territory to reach a certain system quickly."
 
+outfit "Loading Priority Marker"
+	category "Special"
+	cost 500
+	thumbnail "outfit/license"
+	"preferred transporter" 1
+	description "This transponder signals that the vessel it is installed in is preferred for transporting passengers and cargo. Spaceport crews will board passengers and load cargo into such ships before the other ones that belong to the same owner. Priority markers allow fleet captains to reduce the risk of losing passengers or valuable cargo by only marking their strongest or most-protected ships as preferred transports."
```

Comment:
`the vessel it is installed in is preferred` -> `the vessel on which it is installed is preferred`